backend/apis/clients/list-clients.api.js

```diff
@@ -1,23 +1,42 @@
-const { app, databaseError, pool, invalidRequiest } = require("../../server");
+const { app, databaseError, pool, invalidRequest } = require("../../server");
 const mysql = require("mysql");
 
 app.get("/api/clients", (req, res, next) => {
   pool.getConnection((err, connection) => {
     if (err) {
       return databaseError(req, res, err);
     }
-    var queryString = "SELECT ? AS result";
 
-    const query = mysql.format(queryString, [req.query.foo]);
+    var queryString =
+      "SELECT cl.id,cl.firstName,cl.lastName,cl.birthday,ct.zip,ct.primaryPhone,cl.addedBy as addedById,us.firstName as addedByFirstName,us.lastname as addedByLastName,cl.dateAdded FROM clients cl JOIN contactInformation ct ON cl.id = ct.clientId JOIN users us ON cl.addedBy = us.Id WHERE (cl.firstName LIKE ? AND cl.lastName LIKE ?) OR ct.zip = ? ORDER BY cl.firstName,cl.lastName DESC LIMIT 100";
```

Comment:
This is pretty difficult to read -- a template string with multiple lines would be easier. See related https://github.com/JustUtahCoders/comunidades-unidas-internal/pull/77/files#r285938028

Reply:
I had already fixed this from your earlier feedback but I broke it yesterday..